pandas/io/excel/_odswriter.py

```diff
@@ -58,6 +58,16 @@ def __init__(
         self.book = OpenDocumentSpreadsheet(**engine_kwargs)
         self._style_dict: dict[str, str] = {}
 
+    @property
+    def sheets(self) -> dict[str, Any]:
```

Comment:
is there a way to type these more specifically?

Reply:
Not easily, as far as I can tell. I plan to look into this more in the future.

Regarding whatsnew - this is not user facing. Currently in docs no attribute of ExcelWriter is public. I'd like to refactor ExcelWriter (namely, making protected attributes start with `_`) before updating docs.

Reply:
got it totally fine (make issues if you cannot to get to it soon), otherwise PR's totally fine.

> is there a way to type these more specifically?

can we use the base class generic for typing? (again nbd)